ui/app/src/state/epics/content.ts

```diff
@@ -201,6 +202,31 @@ const content: CrafterCMSEpic[] = [
       })
     ),
   // endregion
+  // region conditionallyUnlockItem
+  (action$, state$) =>
+    action$.pipe(
+      ofType(conditionallyUnlockItem.type),
+      withLatestFrom(state$),
+
+      filter(
+        ([{ payload }, state]) =>
+          !state.content.itemsByPath[payload.path].lockOwner ||
+          state.content.itemsByPath[payload.path].lockOwner === state.user.username
```

Comment:
Why would you issue an unlock call for an item that's already unlocked?
```suggestion
        ([{ payload }, state]) =>
          state.content.itemsByPath[payload.path].lockOwner === state.user.username
```